edx_credentials_themes/__init__.py

```diff
@@ -1 +1 @@
-__version__ = '0.1.9'
```

Comment:
Sanity check: why not `0.1.10`?

Reply:
Because we missed a version bump last time